packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts

```diff
@@ -20,12 +20,13 @@ import { FabricEnvironmentRegistryEntry } from '../registries/FabricEnvironmentR
 import * as fs from 'fs-extra';
 import * as path from 'path';
 import { SettingConfigurations } from '../../configurations';
-import { FabricNode } from '../fabric/FabricNode';
+import { FabricNode, FabricNodeType } from '../fabric/FabricNode';
 import { FabricEnvironment } from '../fabric/FabricEnvironment';
 import { ExtensionCommands } from '../../ExtensionCommands';
 import { FileSystemUtil } from '../util/FileSystemUtil';
 import Axios from 'axios';
 import { ExtensionUtil } from '../util/ExtensionUtil';
+import https = require('https');
```

Comment:
this should be imported like other node modules

Reply:
Addressed.